stock_lot_scrap/views/sock_scrap_lot_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+    <record id="view_production_lot_form" model="ir.ui.view">
+        <field name="name">stock.production.lot.scrap.form</field>
+        <field name="model">stock.production.lot</field>
+        <field name="inherit_id" ref="stock.view_production_lot_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@class='oe_button_box oe_right']" position="before">
+                <header>
```

Comment:
This defines a new header because the view doesn't contain it, but it can be define in another module (like the one that blocks lots). You can overwrite fields_view_get and see if there's already a header, and if not, create it. See this code for something similar: https://github.com/OCA/crm/pull/38/files#diff-df883f43bfa35a2a786daa20cc741e74R27